cypress/integration/settings-spec.js

```diff
@@ -33,9 +36,28 @@ describe('settings page', () =>{
   })
 
   it('only the first element should be selected on load', () => {
-    firstElementOfTheSettingsMenuList.should('have.attr', 'nav-selected', 'true')
```

Comment:
for some reason this "should" was not asserting anything, I couldn't use firstElementOfTheSettingsMenuList here but on the next line it works.
if anyone can find the "why" that would be great.

Reply:
it works from my side, does it work as well in CI?

Reply:
the test passes but the assert is never done.
you can see by running `npx cypress open` on master branch and run that test.
there is no asserts being run.
![image](https://user-images.githubusercontent.com/5482105/71730162-c15beb00-2e41-11ea-8b64-8f624e0a76b2.png)